front/src/utils/timeManipulation.ts

```diff
@@ -16,6 +16,10 @@ export function ms2sec(ms: number) {
   return ms / 1000;
 }
 
+export function ms2min(ms: number) {
```

Comment:
nit: I prefer msToMin
```suggestion
export function msToMin(ms: number) {
```

Reply:
The function name is in harmony with the file
![Screenshot From 2025-02-14 09-22-13](https://github.com/user-attachments/assets/6543c80f-8fb0-4cd9-8381-aec80b0a02d9)

Reply:
Note, we have some duplication between `physics.ts` (uses `XToY` style) and `timeManipulation.ts` (uses `X2Y` style). Would be nice to unify these, but it's orthogonal to this PR.

Reply:
Fine, we can refacto this in a future PR

Reply:
Okay noted on my side